### ISSUE TITLE
Distinguish update / new entry

### ISSUE THREAD

**Issue by tobiasKaminsky** [nickv-nextcloud/event_update_notification#12]
For both update / new calendar entry, I do get the same notification.
It should show update instead of new when changing an existing event.

**Comment by nickvergessen** [comment]
> ![Bildschirmfoto von 2019-11-25 12-45-38](https://user-images.githubusercontent.com/213943/69538015-a4e6ac00-0f81-11ea-8c50-16f576c94b9d.png)

Works fine here. Maybe a translation problem?

**Comment by tobiasKaminsky** [comment]
indeed, it works. I guess it was either not working in a very first version, or I mixed it up.
Thanks for the nice app!

**Comment by tobiasKaminsky** [comment]
We had this again and traced it down to repeating events.
So if you have an appointment that is every week, change/delete only one of them, you do not receive the full info.

**Comment by nickvergessen** [comment]
So by now this works as good as it gets. It shows "X updated Y in Z" when changing a single occurrence and also when deleting a single occurrence.
Further information what changed is unluckily not available in the event a the moment.
